src/containers/Chat/Chat.tsx

```diff
@@ -1,14 +1,24 @@
 import { useState } from 'react';
 import { Loading } from 'components/UI/Layout/Loading/Loading';
-import { chatRoutes } from 'routes/AuthenticatedRoute/AuthenticatedRoute';
+import { chatRoutes, groupRoutes } from 'routes/AuthenticatedRoute/AuthenticatedRoute';
 import { ChatSubscription } from './ChatSubscription/ChatSubscription';
+import { GroupMessageSubscription } from 'containers/WA_Groups/GroupMessageSubscription';
+import { useLocation } from 'react-router';
 
 export const Chat = () => {
   const [loaded, setDataLoaded] = useState(false);
+  const location = useLocation();
+  let isWAGroup: boolean = location.pathname.includes('group');
+  let routes: any = chatRoutes;
+  if (isWAGroup) {
```

Comment:
Can you check if we can move this logic to the routes and create a new component for this